src/components/EventMap.js

```diff
@@ -14,6 +16,8 @@ export default function(store){
     data: {
       mapRef: null,
       eventsLayer: null,
+      initialCoordinates: [-96.9 , 37.8],
+      initialZoom: 4
```

Comment:
oh woah... @stuartsan are your pin icons still that big if you update this branch? That is NOT the intended size lol.